stories/cli/test.fmf

```diff
@@ -47,9 +47,24 @@ story: 'As a user I want to comfortably work with tests'
     documented: /docs/examples#convert-tests
 
 /export:
-    story: 'I want to export test metadata into desired format.'
-    example: tmt test export --format yaml
-    implemented: /tmt/base
+    story: 'I want to export test metadata.'
+
+    /format:
+        story: 'I want to export test metadata into given format.'
+        example: tmt test export --format yaml
+        implemented: /tmt/base
+
+    /nitrate:
+        story: 'I want to export test metadata into nitrate.'
+        example: tmt test export --nitrate
+        description: |
+            In order to keep metadata in sync with the old test
+            case management system we need to export selected set
+            of attributes (tags, contact, relevancy) back to
+            nitrate. The full fmf object identifier should be
```

Comment:
There is going to be actual export of data into tcms? I was under the impression that only thing that is going to be put into tcms is reference to fmf case (as well as note that this case has been ported into fmf).

Reply:
My impression was 'the migration can possibly drag out, and bidirectional synchronisation will ease the transition period'.

Reply:
I think @kkaarreell mentioned in his original request that it would be needed... Yes, found here:

> Update relevant metadata (e.g. relevancy) in the respective TCMS test case (and ideally be able to keep that in sync in the future). This is important to preserve TCMS based filtering functional.

Which makes sense to me. For example to be able to easily filter using tags in the webui. Until we are completely liberated from the tool... which will take some time.

Reply:
This seems... problematic to me.
I assume that once test case is ported into fmf, the fmf will become primary source of information (that is potential conflict in data means fmf is right, in another words changes just in tcms will be erased by next `tmt test export` - this won't go well with QEs I think).

Also it means we will have to extract way more information from tcms when using `tmt test convert` (all of them really) - this isn't a problem per se, however it might lead to fmf files being cluttered with all the information. Not to mention all the naming problems which were already encountered (tcms summary != fmf summary).

Reply:
Let's make one point clear: I think the syncing should not be bidirectional. We should only synchronize any changes to fmf metadata to nitrate. Once the test case metadata are migrated the only source of truth would be (ideally) fmf metadata. (For now, intentionally omitting the question of detailed hardware requirements which is a separate story.) Would that make more sense, @hegerj?

Reply:
Do I understand correctly: The need to export data is 
- For tooling which finds cases directly by 'tags' or 'summary__contains' (or other fields)
- To allow filter for tcms-plan, tcms-run if we do not want to make them fmf aware (which shouldn't be that difficult).

Is there am agreement on doing just one way sync from FMF metadata to Nitrate test case?
Could we make clear decisions about source of truth for each nitrate test case attribute?
- Sync: yes/no
- Source for sync is fmf metadata attribute:

Most of the fields are not "converted" at all to fmf, so user might be lost about what is happening.

Nitrate test case fields:
```
Default Tester
Product
Category
Estimated Time
Priority
Status
Alias
Author
Automated
Requirement
Script
Arguments
Reference Link
Notes

Attachments (tab)
Components (tab)
Test Plans (tab)
Tags (tab)
Bugs (tab)
```

In Structured Field we definitely store (but user could create custom values)
```
hardware
relevancy
pepa
purpose-file
description
```

Reply:
Yes, the motivation is to allow old tools which are not yet `fmf` aware or capable to work with the metadata such as `tags` directly from nitrate, plus also to allow users browse and filter test cases using web interface. Perhaps @kkaarreell has some more thoughts on this.

Regarding the list of attributes: Yes, I think it is a good idea to list all attributes which should be synced back. Will update the pull request with it.

Reply:
List of attributes added, including mapping.